### PR TITLE
Update bindings

### PR DESCRIPTION
**Pull request by jmercouris**
I've made a branch, please feel free to push changes to the bindings you think are necessary.

### CODE REVIEW

**Comment by jmercouris** [comment]
At the end we can rebase everything into a single commit...!

**Comment by jmercouris** [comment]
I can't think of much else to change, I've been through most of the keybindings and I have no ideas. I set them to what I thought were sensible defaults at the time of creation.

File: source/base-mode.lisp [hunk 1]
```diff
@@ -66,11 +64,13 @@ This mode is a good candidate to be passed to `make-buffer'."
...
                     (list
                      "C-x C-c" 'quit
-                     "C-x k" 'delete-buffer ; Emacs' default behaviour is to query.
+                     "C-x k" 'delete-buffer
+                     "C-x C-k" 'delete-current-buffer
                      "C-x left" 'switch-buffer-previous
                      "C-x right" 'switch-buffer-next
                      "C-x b" 'switch-buffer
                      "C-x C-b" 'list-buffers
+                     "C-x C-b" 'list-buffers
```
Comment (Ambrevar) [inline source/base-mode.lisp:73]: Dupplicate?

File: source/base-mode.lisp [hunk 2]
```diff
@@ -13,11 +13,9 @@ This mode is a good candidate to be passed to `make-buffer'."
                      "C-q" 'quit
                      "C-[" 'switch-buffer-previous
                      "C-]" 'switch-buffer-next
-                     "C-x b" 'switch-buffer
-                     "C-x C-b" 'list-buffers
-                     "C-x k" 'delete-buffer
+                     "M-[" 'switch-buffer
+                     "M-]" 'switch-buffer
```
Comment (Ambrevar) [inline source/base-mode.lisp:17]: Not sure about this one, it does not seem intuitive.  What about `C-\`?  It's often next to `[` or `]`.

Comment (jmercouris) [reply source/base-mode.lisp:17]: I don't see how C-\ is any more intuitive :-D

File: source/base-mode.lisp (hunk 1 above)
Comment (jmercouris) [reply source/base-mode.lisp:73]: Duplicate!

File: source/base-mode.lisp (hunk 2 above)
Comment (jmercouris) [reply source/base-mode.lisp:17]: This is the one binding I really struggled with, I don't see a parallel in any other CUA programs to draw inspiration. Maybe "C-." would be a good one? or "C-/" or "C-?"?

Comment (jmercouris) [reply source/base-mode.lisp:17]: We could take a hint from macOS expose binding which would be "C-down"

Comment (Ambrevar) [reply source/base-mode.lisp:17]: OK for `C-down`.
My thinking behind `C-\` is that it's next to `C-[` and `C-]`.
What I don't like about `M-[` and `M-]` is that the brackets imply a symmetry which isn't there.  `\` does not suffer from this (unless you consider the symmetry of `\` to be `/`).

Comment (jmercouris) [reply source/base-mode.lisp]: I had to switch to "M-down" to avoid collision with scroll to bottom of page.

**Comment by jmercouris** [comment]
@aartaka any other suggestions? @aadcg ? @Ambrevar ?

**Comment by aadcg** [comment]
Nothing worth sharing. All good for me.

**Comment by aartaka** [comment]
No suggestions from me — it seems all good :)

File: source/base-mode.lisp [hunk 3]
```diff
@@ -60,13 +57,14 @@ This mode is a good candidate to be passed to `make-buffer'."
                      "C-/" 'reopen-buffer
                      "C-shift-t" 'reopen-buffer
                      "C-T" 'reopen-buffer
-                     "C-p" 'print-buffer
-                     "M-i" 'focus-first-input-field)
```
Comment (Ambrevar) [inline source/base-mode.lisp:64]: I think this M-i binding should stay, no?

File: source/base-mode.lisp [hunk 4]
```diff
@@ -60,13 +57,14 @@ This mode is a good candidate to be passed to `make-buffer'."
...
+                     "C-p" 'print-buffer)
 
                     scheme:emacs
                     (list
+                     "M-i" 'focus-first-input-field
```
Comment (Ambrevar) [inline source/base-mode.lisp:64]: And I'd remove it from here.

File: source/base-mode.lisp (hunk 2 above)
Comment (Ambrevar) [reply source/base-mode.lisp]: OK.

File: source/base-mode.lisp (hunk 3 above)
Comment (jmercouris) [reply source/base-mode.lisp:64]: I moved it to the emacs section, because it seems an emacs like binding a CUA user would not use

Comment (jmercouris) [reply source/base-mode.lisp:64]: I tried M-i in Firefox and it did nothing

**Comment by Ambrevar** [comment]
True, it's not a CUA binding, but it's not an Emacs binding either.

I suggest the following approach: set all Nyxt specific bindings in the
CUA scheme since the other schemes inherit from CUA.

**Comment by jmercouris** [comment]
OK, I will change that and then merge.

File: source/base-mode.lisp (hunk 3 above)
Comment (aadcg) [reply source/base-mode.lisp:64]: > I tried M-i in Firefox and it did nothing

But that wouldn't be a good analogy anyway. Inspiration would have to come from browser extensions like tridactyl (that I have never uses and know nothing about).

I seem to agree with @Ambrevar.

**Pull request merged**